leapp/cli/upgrade/__init__.py

```diff
@@ -175,6 +175,40 @@ def upgrade(args):
         sys.exit(1)
 
 
+@command('preupgrade', help='Generate preupgrade report')
+def preupgrade(args):
+    if os.getuid():
+        raise CommandError('This command has to be run under the root user.')
+
+    context = str(uuid.uuid4())
+    configuration = {
+        'debug': os.getenv('LEAPP_DEBUG', '0'),
+        'verbose': os.getenv('LEAPP_VERBOSE', '0'),
+    }
```

Comment:
I meant, to have whitelist_experimental here.

Reply:
Good point, I'll add it.

Reply:
Now that you mention it, would it make sense to implement `--resume` as well or not?

Reply:
Well not really, because resume was used for continuation after reboot or planned interruption - I don't see why that would be needed for preupgrade